website/content/en/docs/Installation/_index.md

```diff
@@ -56,5 +56,11 @@ To install, you need only to type these commands:
 
 ```bash
 $ helm repo add jenkins https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart
-$ helm install jenkins/jenkins-operator
+$ helm install <name> jenkins/jenkins-operator
+```
+
+To add custom labels and annotations, you can use `values.yaml` file or pass them into `helm install` command, e.g.:
+
+```bash
+$ helm install <name> jenkins/jenkins-operator --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue
```

Comment:
`helm install <name> jenkins/jenkins-operator --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue` -> `helm install <name> jenkins/jenkins-operator`. 

Add separated section with info about `values.yaml` customization possibility.

Reply:
Since I'm already changing docs later (recommiting to fix_docs pull request) I guess I can add it then to not generate docs twice? We could do separate page about customizing with helm and leave a link there or add a 'helm section' in existing customization page.